spec/ordinary-and-exotic-object-behaviours.html

```diff
@@ -49,14 +46,9 @@ <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
         <emu-alg>
           1. Assert: IsPropertyKey(_P_) is *true*.
           1. If Type(_P_) is a Symbol, return *false*.
-          1. If _Desc_.[[Writable]] is present and has value *true*, return *false*.
-          1. If _Desc_.[[Enumerable]] is present and has value *false*, return *false*.
-          1. If _Desc_.[[Configurable]] is present and has value *true*, return *false*.
           1. If IsAccessorDescriptor(_Desc_) is *true*, return *false*.
-          1. Let _current_ be ! RecordGet(_R_, _P_).
-          1. If _current_ is ~empty~ return *false*.
-          1. If _Desc_.[[Value]] is present, return ! SameValue(_Desc_.[[Value]], _current_).
-          1. Return *true*.
+          1. Let _current_ be ! RecordGetOwnProperty(_R_, _P_).
```

Comment:
```suggestion
          1. Let _current_ be ! RecordGetOwnProperty(_R_, _P_).
          1. If _current_ is *undefined*, return *false*.
```
just to have an explicit suggestion :-)